app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -89,4 +89,5 @@
     <string name="drawer_draw_badge">画剪贴画</string>
     <string name="clipart_saved_success">ClipArt已成功保存</string>
     <string name="clipart_saved_error">保存剪贴画时出错</string>
+    <string name="drawer_saved_cliparts">Saved Cliparts</string>
```

Comment:
Shouldn't it be in chinese here?

Reply:
Yes, but I usually batch translate before releasing a version on playstore xD.

Reply:
Ok then :)

Reply:
Change or remove

Reply:
@iamareebjamal changing